sdk/videoanalyzer/video-analyzer-edge/test/payloadCreation.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { assert } from "chai";
+import {
+  PipelineTopology,
+  RtspSource,
+  UnsecuredEndpoint,
+  NodeInput,
+  IotHubMessageSink,
+  createRequest
+} from "../src";
+
+describe("test", () => {
```

Comment:
Are there any plans to add a test that sends the request and verifies the response?

Reply:
Since our sdk relies on the Azure IoT Hub sdk to actually send the requests we cannot do a live test for it